src/main/perf/LineFileDocs.java

```diff
@@ -410,6 +422,9 @@ public static int idToInt(String id) {
       body = new Field("body", "", bodyFieldType);
       doc.add(body);
 
+      randomLabel = new Field("randomLabel", "", StringField.TYPE_STORED);
```

Comment:
Hmm why did we also need a stored field?  I can understand `SORTED` DV (so we can test DV faceting as well)...